### PR TITLE
P 990 benchmark fix

### PR DESCRIPTION
**Pull request by wangminqi**
(1) Fix bridge related benchmark impl
(2) add several skipped benchmark back
(3) add cumulus fix in 0.9.43 update for frame-system benchmark fix
https://github.com/paritytech/cumulus/pull/2766

### CODE REVIEW

**Review by Kailai-Wang — APPROVED** [review]
Looks good

File: Cargo.toml [hunk 1]
```diff
@@ -3,10 +3,11 @@ resolver = "2"
...
     'node',
     'pallets/account-fix',
-    'pallets/assets-handler',
     'pallets/bitacross',
-    'pallets/bridge',
-    'pallets/bridge-transfer',
+    'pallets/bridge-related-pallets/assets-handler',
+    'pallets/bridge-related-pallets/bridge',
+    'pallets/bridge-related-pallets/bridge-transfer',
+    'pallets/bridge-related-pallets/common',
```
Comment (Kailai-Wang) [inline Cargo.toml]: I prefer this (directory) naming:

```
bridge-related-pallets => bridge
pallets/bridge-related-pallets/bridge => pallets/bridge/chain-bridge # to be consistent with runtime name ChainBridge
```

**Comment by wangminqi** [comment]
By the way, although CI passed PR approved, the merge eligibility is up to its benchmark passing.
Please do not merge before that.

File: Cargo.toml (hunk 1 above)
Comment (Kailai-Wang) [reply Cargo.toml]: I'll try to submit a PR to your branch

**Comment by wangminqi** [comment]
https://github.com/litentry/litentry-parachain/actions/runs/10607548866/job/29402295958
benchmark fine. Seems failed to submit a PR.

**Pull request merged**